board/piksiv3/post_image.sh

```diff
@@ -19,14 +19,14 @@ generate_prod() {
   local CFG=piksiv3_$HW
   local UBOOT_BUILD_DIR=$UBOOT_DIR/build/${CFG}_prod
   local OUTPUT_DIR=$BINARIES_DIR/${CFG}_prod
-  local BR_GIT_VERSION=$(git -C $BR2_EXTERNAL rev-parse --short HEAD)
+  local BR_GIT_VERSION=$(git describe --tags --dirty --always --match 'v[0-9]\.[0-9]')
```

Comment:
you need the `-C $BR2_EXTERNAL` as this script is not executed from the desired git directory